Dockerfile

```diff
@@ -1,23 +1,23 @@
-FROM ruby:2.5.3
+FROM ruby:2.5.3-alpine
+RUN apk add --update tzdata && \
+    cp /usr/share/zoneinfo/Europe/London /etc/localtime && \
+    echo "Europe/London" > /etc/timezone
 
-RUN apt-get update && apt-get install apt-transport-https
-
-RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
-  && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
-  && curl -sL https://deb.nodesource.com/setup_8.x | bash -
-
-RUN apt-get update && \
-  apt-get install -y \
-  yarn \
-  nodejs
+RUN apk add --update --virtual runtime-dependances \
+ yarn \
+ openssl-dev
 
 ENV APP_HOME /app
 RUN mkdir $APP_HOME
 WORKDIR $APP_HOME
 
 ADD Gemfile $APP_HOME/Gemfile
 ADD Gemfile.lock $APP_HOME/Gemfile.lock
-RUN bundle install
+
+RUN apk add --update --virtual build-dependances \
+ build-base  && \
+ bundle install --jobs=4 && \
+ apk del build-dependances
```

Comment:
Does using `--virtual` mean that you don't need to clear the cache/use `--no-cache`?

Reply:
https://stackoverflow.com/questions/49118579/alpine-dockerfile-advantages-of-no-cache-vs-rm-var-cache-apk